### DISCUSSION TITLE
Static Container

### DISCUSSION THREAD

**Discussion by emdadgar2** [dadhi/DryIoc#423] [Q&A] [answered]
Hi,
In MVC 5 and DryIoc.Web.dll, we can use created Container in application startup,
we could reference it to an static variable and use everywhere in application (even another app. layer)
and InWebRequest reuse worked fine on it. 

In NET 5 or NetCore version, static reference to Container  will failed with error: "code: Error.NoCurrentScope"

Is there any way to access a static Container to use in another layres?

Thanks anyway

**Comment by dadhi** [comment]
.Net Core and .Net 5+ moved away from the shared (static) state and imho you should too.

But if you really want to configure the container with `AsyncExecutionFlowScopeContext`.

**Comment by dadhi** [comment, marked as answer]
Here is the docs https://github.com/dadhi/DryIoc/blob/master/docs/DryIoc.Docs/ReuseAndScopes.md#scopecontext
